eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -511,4 +507,15 @@ function CheckArtifactShaAgainstTagsList($priorExistingTagList, $releaseSha, $ap
     Write-Host "Tags already existing with different SHA versions. Exiting."
     exit(1)
   }
+}
+
+# given the package id and package version, generate the release tag
+
+function GenerateReleaseTag($packageId, $packageVersion) {
```

Comment:
While this function might help a little it doesn't give us the same tag because we generate packageId and packageVersion differently between the upload-blobs and other usage. I think we might need to call the VerifyPackage function to get the same tag.